gh-actions/common/build-debian/action.yml

```diff
@@ -140,6 +147,9 @@ runs:
           fi
           echo "::endgroup::"
 
+          # FIXME: This part should moved to ${{ inputs.extra-source-build-script }}
+          # Since it's very project-specific, as most of projects don't care about
+          # Having git at all when building the package source.
```

Comment:
```suggestion
          # Since it's very project-specific, as most projects don't care about
          # having git at all when building the package source.
```